src/distances.jl

```diff
@@ -3,13 +3,21 @@
 # ------------------------------------------------------------------
 
 """
-    Ellipsoidal(semiaxes, angles)
+    Ellipsoidal(semiaxes, angles; convention=:TaitBryanExtr)
 
 A distance defined by an ellipsoid with given `semiaxes` and rotation `angles`.
+Default angle convention is Tait-Bryan ZXY, extrinsic right-handed rotation
+Other conventions available: :TaitBryanIntr, :EulerExtr, :EulerIntr,
+                             :GSLIB, :Leapfrog, :Datamine
+* :EulerExtr, :EulerIntr and :Datamine assumes ZXZ rotation sequence
```

Comment:
Could you create a section in the docstring with `## Conventions` and then add each convention as a bullet? It is awesome that you added support to so many conventions, including those from popular commercial software :100:

Reply:
Done, see if it's okay